consensus/src/selection/handler.rs

```diff
@@ -17,7 +17,7 @@ use tokio::sync::Mutex;
 
 pub struct Selection<D: Database> {
     pub(crate) db: Arc<Mutex<D>>,
-    pub(crate) _sv_registry: SafeStepVotesRegistry,
+    pub(crate) _sv_registry: SafeCertificateInfoRegistry,
```

Comment:
Can this be removed?

Reply:
It can be removed but I put it for consistency with other steps.